Compiler/VisitorBase.cs

```diff
@@ -91,11 +91,15 @@ private T d
         {
             get
             {
+#pragma warning disable CS8653 // A default expression introduces a null value for a type parameter.
```

Comment:
Is there no better way of resolving this?

Reply:
Maybe, not really sure atm